fandor/react-native-video
# Feature/fairplay

#### Purpose of Pull Request?
Add Fairplay DRM support to the React-Native App

#### Provide link to story:
https://trello.com/c/pzLfQCzC/32-add-fairplay-support-to-ios

#### How should this code be validated? 
Code should follow our coding standards

#### QA Steps
_Note:  This is currently pointed at staging but if you want to test in production change the static 
variables in the `RCTVideoResourceLoaderDelegate.m` file._

- Add this branch to the ios repo package.json
  -  "react-native-video": "https://github.com/fandor/react-native-video#feature/fairplay"
- Run `npm i` to install this branch
- Side load the app from Xcode (If you need help with this let me know)
- Make sure you are logged in
- Play a film that has been DRM'd (Make sure it has the DRM flag checked in the admin area)
  - You can search for Zyara with Dei el Ayoubi and can be used to test.
- The film should play.
